internal/gensupport/resumable.go

```diff
@@ -85,6 +85,9 @@ func (rx *ResumableUpload) doUploadRequest(ctx context.Context, data io.Reader,
 	invocationHeader := fmt.Sprintf("gccl-invocation-id/%s gccl-attempt-count/%d", rx.invocationID, rx.attempts)
```

Comment:
Out of curiosity, will this extra `gccl-invocation-id` key-value pair still be necessary in `X-Goog-Api-Client` if there is a purpose-built header now? If not, please file an issue and add it as a todo to remove this later.